src/test/java/core/basesyntax/TransactionParserServiceTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class TransactionParserServiceTest extends FruitShopTest {
+    private static final int SIZE_OF_DEFAULT_FILE = 8;
+
+    @Test
+    public void createTransaction_ok() {
+        List<String> infoInStrings = readerService.readFileToList(testFile);
+        List<FruitTransaction> actual = transactionParser.createTransaction(infoInStrings);
```

Comment:
You can create these objects in class (not in method) and initialize these in method @BeforeClass